lib/metasploit/framework/mssql/client.rb

```diff
@@ -1,15 +1,57 @@
-require 'metasploit/framework/tcp/client'
 require 'metasploit/framework/mssql/tdssslproxy'
-require 'metasploit/framework/mssql/base'
 
 module Metasploit
   module Framework
     module MSSQL
 
       module Client
         extend ActiveSupport::Concern
-        include Metasploit::Framework::Tcp::Client
-        include Metasploit::Framework::MSSQL::Base
```

Comment:
I'm not sure if this PR is required, it looks like this mixin is valid to use 👀 

From our previous call - the goal should be to consolidate the two implementations of the MSSQL login functionality into one place first, before creating a concrete client class 👀 

ie. the Login scanner, versus the `remote/mssql.rb` implementation